apps/web/ui/modals/add-edit-link-modal/index.tsx

```diff
@@ -363,7 +363,7 @@ function AddEditLinkModal({
         }
       }}
     >
-      <div className="scrollbar-hide grid max-h-[95vh] w-full divide-x divide-gray-100 overflow-auto md:grid-cols-2 md:overflow-hidden">
+      <div className="scrollbar-hide grid max-h-[95dvh] w-full divide-x divide-gray-100 overflow-auto md:grid-cols-2 md:overflow-hidden">
```

Comment:
`dvh` seems to do a better job here so that the max height is consistent relative to the actual viewport - we should try to test this out some more in different devices/browsers.

Reply:
good idea!